### PR TITLE
ODP-2410  Upgraded jquery and bootstrap into 3.7.1 and 5.3.3 respectively

### PR DESCRIPTION
**Pull request by senthh**
Upgraded jquery and bootstrap into 3.7.1 and 5.3.3 respectively

### CODE REVIEW

**Comment by senthh** [comment]
Test Cluster : http://ce14.acceldata.dvl:8088/cluster/cluster

Reference for comparison: http://ce15.acceldata.dvl:8088/cluster/cluster

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-ui/src/main/webapp/bower.json [hunk 1]
```diff
@@ -8,16 +8,16 @@
...
-    "jquery-ui": "1.11.4",
+    "jquery-ui": "1.12.1",
     "moment": "2.12.0",
     "moment-timezone": "0.5.1",
     "more-js": "0.8.2",
-    "bootstrap": "3.3.6",
+    "bootstrap": "5.3.3",
     "d3": "~3.5.6",
-    "datatables": "~1.10.8",
+    "datatables": "~1.10.7",
```
Comment (prabhjyotsingh) [inline hadoop-yarn-project/hadoop-yarn/hadoop-yarn-ui/src/main/webapp/bower.json]: Any specific reason to downgrade datatables?

Comment (senthh) [reply hadoop-yarn-project/hadoop-yarn/hadoop-yarn-ui/src/main/webapp/bower.json]: We use only dt-1.10.7  version in the actual code, but in, bower it was mentioned as 1.10.8; 
And I tried to upgrade this  into dt-1.10.8 in code also, but other components(like jquery 3.7.1) were not compatible with dt-1.10.8. 
So in order to make consistency in both code and bower,  we have downgraded to dt-1.10.7 in bower.json too

**Review by prabhjyotsingh — APPROVED** [review]
+1

**Pull request merged**